addons/elasticsearch/es-settings.json

```diff
@@ -41,6 +41,13 @@
           "lowercase"
         ]
       },
+      "atlan_space_analyzer": {
+        "type": "custom",
+        "tokenizer": "whitespace",
+        "filter": [
```

Comment:
We don't lowercase filter and what does apostrophe filter do?

Reply:
Apostrophe filter removes all characters after an apostrophe, including the apostrophe itself. If we use atlan_space_analyzer only while searching guids then these filters are not needed though.